### ISSUE TITLE
Book(s) skeleton

### ISSUE THREAD

**Issue by HelenaSabel** [ebeshero/Amadis-in-Translation#19]
This is a non-pressing issue.
We have a TEI file for each chapter of each version, but I think we need a main "index" file.
I'm not sure if the best way would be to think of Amadis as a whole and have in the same file the references to all the chapters (both Southey's and Montalvo's) or to create a file for each one of them, mainly, a `<div type="book">` that would contain all the chapters heads and their referenced IDs.
In any case, I'm also using this issue to remind us that the "book argument" in the Montalvo's version wasn't transcribed because, from a hierarchical point of view, I couldn't include it under chapter 0.

**Comment by ebeshero** [comment]
Good idea, @HelenaSabel! We'll start by making a TEI file of it, and then we can transform it so it's a central page for our project website as that comes together.

And we could then include introductory material in both Montalvo and Southey. (We should at the very least prepare a file to contain Southey's preface, since that's where he describes his translation practice.)
